tests/e2e/runAnalysis.spec.ts

```diff
@@ -41,17 +45,44 @@ test("Can request a scenario analysis run", async ({ page, baseURL, headless })
+  await expect(page.locator("#prevalence-container .highcharts-plot-band")).toBeVisible();
+  await expect(page.locator("#prevalence-container").getByLabel("View chart menu, Chart")).toBeVisible();
+
+  await expect(page.locator("#hospitalised-container")).toBeVisible();
+  await page.locator("#hospitalised-container").scrollIntoViewIfNeeded();
+  await expect(page.locator("#hospitalised-container .highcharts-xaxis-labels")).toBeVisible();
+  await expect(page.locator("#hospitalised-container .highcharts-yaxis-labels")).toBeVisible();
+  await expect(page.locator("#hospitalised-container .highcharts-plot-band")).toBeVisible();
+  await expect(page.locator("#hospitalised-container .highcharts-plot-line")).toBeInViewport();
+  await expect(page.locator("#hospitalised-container").getByLabel("View chart menu, Chart")).toBeVisible();
+
+  await expect(page.locator("#dead-container")).toBeVisible();
+  await page.locator("#dead-container").scrollIntoViewIfNeeded();
+  await expect(page.locator("#dead-container .highcharts-xaxis-labels")).toBeVisible();
+  await expect(page.locator("#dead-container .highcharts-yaxis-labels")).toBeVisible();
+  await expect(page.locator("#dead-container").getByLabel("View chart menu, Chart")).toBeVisible();
+
+  await expect(page.locator("#vaccinated-container")).toBeVisible();
+  await page.locator("#vaccinated-container").scrollIntoViewIfNeeded();
+  await expect(page.locator("#vaccinated-container .highcharts-xaxis-labels")).toBeVisible();
+  await expect(page.locator("#vaccinated-container .highcharts-yaxis-labels")).toBeVisible();
+  await expect(page.locator("#vaccinated-container").getByLabel("View chart menu, Chart")).toBeVisible();
+
   // To regenerate these screenshots:
   // 1. Insert a generous timeout so that screenshots are of the final chart, not the chart half-way through
   //    its initialization animation: `await page.waitForTimeout(10000);`
   // 2. Delete the screenshots directory, ./<this-file-name>-snapshots
   // 3. Run the tests with `npm run test:e2e` to regenerate the screenshots - tests will appear to fail the first time.
   // Make sure to stop any local development server first so that Playwright runs its own server, in production mode, so that the
   // Nuxt devtools are not present in the screenshots.
-  if (headless) {
-    await expect(page.locator(".accordion-body").first()).toHaveScreenshot("first-time-series.png", { maxDiffPixelRatio: 0.04, timeout: 15000 });
-    await expect(page.locator(".accordion-body").nth(1)).toHaveScreenshot("second-time-series.png", { maxDiffPixelRatio: 0.04 });
-    await expect(page.locator(".accordion-body").nth(2)).toHaveScreenshot("third-time-series.png", { maxDiffPixelRatio: 0.04 });
+  if (headless && useVisualScreenshotTesting) {
+    await expect(page.locator(".highcharts-background").first()).toHaveScreenshot("first-time-series.png", { maxDiffPixelRatio: 0.04, timeout: 15000 });
```

Comment:
I think these changes to the e2e test appear here because the branch is out of date with the latest version of jidea-105-14-oct